source/reference/limitations.txt

```diff
@@ -101,9 +101,35 @@ Multiple Clusters
 - Syncing one source cluster to many destination clusters is not
   supported.
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If the user issues a :dbcommand:`dropDatabase` command on the source cluster,
+this change is not direclty applied on the destination cluster. Instead,
+{+c2c-product-name+} drops user collections and views in the destination 
+database, but it does not drop system collections on the destination database.
+
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created  
```

Comment:
[case suggestion]

Consider using the present tense. 

https://www.mongodb.com/docs/meta/style-guide/writing/use-present-tense/

Reply:
Fixed